Nodejs/Setup/BuildRelease.ps1

```diff
@@ -243,7 +243,9 @@ $managed_files = (
     "Microsoft.NodejsTools.Npm.dll",
     "Microsoft.NodejsTools.TestAdapter.dll",
     "Microsoft.NodejsTools.PressAnyKey.exe",
-    "Microsoft.NodejsTools.VsLogger.dll"
+    "Microsoft.NodejsTools.VsLogger.dll",
+    "Microsoft.NodejsTools.InstallerTelemetryAction.CA",
+    "Microsoft.NodejsTools.InstallerTelemetryAction"
```

Comment:
missing file extensions

Also to make things easier for merging with Naresh's changes, would you mind just moving these above Microsoft.NodejsTools.VsLogger.dll? (he is touching the references to VsLogger in his commit.)